jira7/jira_data_mappings.bal

```diff
@@ -233,27 +233,6 @@ function issueRequestToJson(IssueRequest source) returns json {
     target.fields.project = source.projectId != EMPTY_STRING ? {id:source.projectId} : null;
     target.fields.assignee = source.assigneeName != EMPTY_STRING ? {name:source.assigneeName} : null;
 
-    map<string> sourceMap = {};
-    var value = map<string>.convert(source);
-    if (value is error) {
-        log:printError(<string> value.detail().message);
-    } else {
-        sourceMap = value;
-    }
-
-    sourceMap["summary"] = "";
-    sourceMap["issueTypeId"] = "";
-    sourceMap["projectId"] = "";
-    sourceMap["assigneeName"] = "";
-    
-    foreach var f in sourceMap.keys() {
-        var result = json.convert(sourceMap[f]);
-        if (result.length == 0) {
-            sourceMap[f] = "";
-        } else {
-            target.fields[f] = result;
-        }
-    }
```

Comment:
Why are we removing this code?

Reply:
The IssueRequest record has defined four attribute in the https://github.com/wso2-ballerina/module-jira/blob/master/jira7/jira_types.bal#L326. Here we added the four attribute into the target. Hence no need to do the further configuration. So I have removed.